doc/lua_api.txt

```diff
@@ -7949,8 +7949,9 @@ gets activated (not loaded!)
         -- and not only the first time the block gets activated after the LBM
         -- was introduced.
 
-        action = function(pos, node),
+        action = function(pos, node, dtime_s),
         -- Function triggered for each qualifying node.
+        -- `dtime_s` is the in-game time elapsed since the block was last active
```

Comment:
```suggestion
        -- `dtime_s` is the in-game time (in seconds) elapsed since the block
        -- was last active
```

`dtime_s` suggests seconds anyway, but I think it would be good to clarify.

Reply:
I don't think the `_s` means seconds but I don't know what else either

More importantly, does in-game time even have an unit?

Reply:
Oh, I assumed `_s` meant seconds here.

Game time is in seconds: https://github.com/minetest/minetest/blob/da4a4086cff1c79c5d30d1bb3b7f23076f8dbd17/src/serverenvironment.h#L471-L473